_posts/2023-8-22-introducing-sqlite3-to-kw.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: Introducing SQLite3 to kw
+---
+
+Some months ago, around May, I had the opportunity of helping to introduce a Database Management System (DBMS) to a project that used a file-based database. The DBMS was [SQLite3](https://www.sqlite.org/index.html) and the project was [kw](https://kworkflow.org/). This post describes my experience.
+
+## File-Based Databases
+
+In a quick Google search, I found that file-based databases are also called flat file databases. But what are file-based databases? It's the most naive, but it **can** also be the most agile method of implementing a database on an application.
+
+No matter your level of experience in programming, you probably faced the problem of having to store data persistently. In other words, your application was manipulating data (one can argue that this is the only thing computers do) and you had to store this data not on main memory but on persistent memory, maybe because the application didn't run continuously and it had to access this data on a later execution.
```

Comment:
Maybe change this phrase to something like:
`was manipulating` -> `manipulates`.
`and you had to` -> `and it has to store data in a persistent memory`.